hack/images/dcos/bootstrap.sh

```diff
@@ -22,7 +22,7 @@ echo
 # -C for failing when files are clobbered
 set -ue
 
-if [ "${DEBUG}" == "true" ]; then
+if [ "${DEBUG:-false}" == "true" ]; then
```

Comment:
I'd rather set a default once instead of having to do this everywhere. `: ${DEBUG=false}`